types/Date.php

```diff
@@ -0,0 +1,50 @@
+<?php
+namespace plugin\struct\types;
+
+use plugin\struct\meta\ValidationException;
+
+class Date extends AbstractBaseType {
+
+    protected $config = array(
+        'prefix' => '',
+        'postfix' => '',
```

Comment:
I think we should decide if we always want to support prefix and postfix and if so, move it to AbstractBasetype. Otherwise we should get rid of it where it's not needed. Maybe discuss with Detlef and Stefan